tests/e2e/show_index_info/test_show_index_info.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import sys
+
+import pytest
+from common import Row, cursor, execute_and_fetch_all
+
+
+def test_show_index_info(cursor):
+    index_info = set(execute_and_fetch_all(cursor, "SHOW INDEX INFO;"))
+    expected_index_info = {
+        ("label", "Anatomy", None),
+        ("label", "Gene", None),
+        ("label+property", "Compound", "inchikey"),
+        ("label+property", "Gene", "id"),
+        ("label", "Compound", None),
```

Comment:
shouldn't label index come before label-property index because of 
```
if (type_1 != type_2) {
    return type_1 < type_2;
}
```

Reply:
This test only makes sure that all indices are returned by `SHOW INDEX INFO`. The next test (`test_index_info_sorted`) is about output sorting.

Reply:
Clarified